style.css

```diff
@@ -0,0 +1,170 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+body {
+  background: rgb(230, 227, 227);
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding-left: 80px;
+  padding-right: 80px;
+  height: 57px;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+header nav ul {
+  display: flex;
+  justify-content: center;
+}
+
+header nav ul li {
+  display: block;
+  list-style: none;
+  font-weight: 400;
+}
+
+header nav ul li a {
+  text-decoration: none;
+  padding: 15px 20px;
+  font-size: 20px;
+  color: black;
+}
+
+h1 {
+  font-size: 52px;
+  margin: 40px auto;
+  text-align: center;
+}
+
+.record {
+  width: 60%;
+  height: 100%;
+  margin: 10px auto;
+  padding: 24px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+.element {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  /* margin-bottom: 10px; */
```

Comment:
- [ OPTIONAL ] Kindly avoid leaving commented pieced of code in your codebase. It makes your code unclean :+1:

Reply:
fixed, thanks!